src/App.js

```diff
@@ -1,25 +1,21 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import home from './home'
+import getStarted from './getStarted'
+import results from './results'
 import './App.css';
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            <h1>Welcome to the Team app!!!!</h1>
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+        <Router>
```

Comment:
For simplicity reason, you can unwrap the `<Router>` to surround the `<div className='app'>`.

Hence, you wouldn't need to create the artificial `<div>` underneath to keep a single root element.